cookiecutter.json

```diff
@@ -2,14 +2,16 @@
     "full_name": "Jørgen S. Dokken",
     "email": "dokken@simula.no",
     "github_username": "jorgensd",
-    "project_name": "Package Name",
-    "module_name": "{{ cookiecutter.project_name.lower().replace(' ', '_').replace('-', '_') }}",
+    "project_name": "Research Project 1",
+    "repository_name": "{{ cookiecutter.project_name.lower().replace(' ', '_').replace('-', '_') }}",
+    "module_name": "{{cookiecutter.repository_name}}",
     "project_short_description": "Tools ensuring reproducible and standarized Python packaging.",
     "version": "0.1.0",
     "linting": "y",
     "testing": ["pytest", "pytest-cov", "none"],
     "use_argparse": "y",
```

Comment:
I think it would be nice to have one question first about whether you want a cli and if you want a cli then follow up with a question about argparse (we could also add an option to use `click` or `typer`?)

Reply:
We could do that, But i dont want to add to many options in this PR. Could you make an issue on the subject?

Reply:
See #4